README.md

```diff
@@ -69,89 +70,65 @@ script = {
+def turn_handler(in_request: Message, pipeline: Pipeline) -> Message:
+    # Pass user request into pipeline and get dialog context (message history)
+    # The pipeline will automatically choose the correct response using script
     ctx = pipeline(in_request, 0)
-    # Get last actor response from the context
+    # Get last response from the context
     out_response = ctx.last_response
-    # The next condition branching needs for testing
-    return out_response, ctx
+    return out_response
 
 
 while True:
-    in_request = input("type your answer: ")
-    out_response, ctx = turn_handler(Message(text=in_request), pipeline)
-    print(out_response.text)
+    in_request = input("Your message: ")
+    out_response = turn_handler(Message(text=in_request), pipeline)
+    print("Response: ", out_response.text)
 ```
 
 When you run this code, you get similar output:
 ```
-type your answer: hi
-Okey
-type your answer: Hi
-Hi!!!
-type your answer: ok
-Okey
-type your answer: ok
-Okey
+Your message: hi
+Response:  OK
+Your message: Hi
+Response:  Hi!
+Your message: ok
+Response:  OK
+Your message: ok
+Response:  OK
 ```
 
-To get more advanced examples, take a look at
-[tutorials](https://github.com/deeppavlov/dialog_flow_framework/tree/master/tutorials) on GitHub.
-
-# Context Storages
-## Description
+More advanced examples are available as a part of documentation:
+[tutorials](https://deeppavlov.github.io/dialog_flow_framework/tutorials.html).
 
-Context Storages allow you to save and retrieve user dialogue states
-(in the form of a `Context` object) using various database backends. 
+## Further steps
 
-The following backends are currently supported:
-* [JSON](https://www.json.org/json-en.html)
-* [pickle](https://docs.python.org/3/library/pickle.html)
-* [shelve](https://docs.python.org/3/library/shelve.html)
-* [SQLite](https://www.sqlite.org/index.html)
-* [PostgreSQL](https://www.postgresql.org/)
-* [MySQL](https://www.mysql.com/)
-* [MongoDB](https://www.mongodb.com/)
-* [Redis](https://redis.io/)
-* [Yandex DataBase](https://ydb.tech/)
+To further explore the API of the framework, you can make use of the [detailed documentation](https://deeppavlov.github.io/dialog_flow_framework/index.html). 
+Broken down into several sections to highlight all the aspects of development with DFF,
+the documentation for the library is constantly available online.
 
-Aside from this, we offer some interfaces for saving data to your local file system.
-These are not meant to be used in production, but can be helpful for prototyping your application.
+* [API reference](https://deeppavlov.github.io/dialog_flow_framework/reference.html)
+* [Tutorials: interactive jupyter notebooks](https://deeppavlov.github.io/dialog_flow_framework/tutorials.html)
+* [User guides: code-supplemented instructions](https://deeppavlov.github.io/dialog_flow_framework/user_guides.html)
```

Comment:
What I don't like about the current approach is that we have the index page for documentation with detailed description of sections and these links circumvent that description.

Maybe we should remove these lines (and all the other up until `# Contributing to the Dialog Flow Framework`) so that the only link to the documentation would be to the index page.

Then this section would look like this:
```
## Further steps

To further explore the API of the framework, you can make use of the [detailed documentation](https://deeppavlov.github.io/dialog_flow_framework/index.html). 
Broken down into several sections to highlight all the aspects of development with DFF,
the documentation for the library is constantly available online.

# Contributing to the Dialog Flow Framework
```

Alternatively, we could replace these links with anchored index links (like `index.html#user-guides`).

Personally, I like the first approach better.
Any thoughts?

Reply:
Makes sense to me